mesonbuild/modules/gnome.py

```diff
@@ -1851,8 +1859,9 @@ def mkenums_simple(self, state: 'ModuleState', args: T.Tuple[str], kwargs: 'MkEn
         if body_prefix != '':
             fhead += '%s\n' % body_prefix
```

Comment:
I think you are missing a pragma once here or something given the redefinition error

Reply:
I think it's some kind of computer divinity trying to teach me a lesson. It is exactly after I advocate strongly in favor of `#pragma once` that I hit a case where it failed me. My unit test does `h1 = configure_file(input: 'meson-sample.h', output: 'h1.h', copy: true)` which causes identical header file to have 2 names `h1.h` and `meson-sample.h` and thus gets included twice, where a macro guard would have saved me... And in a unit test I wrote a year ago, it waited all that time to hit me exactly at this moment...

Fixed, there were no reason to copy that header into builddir, we already have `h2.h` that test that case.

Reply:
> I think it's some kind of computer divinity trying to teach me a lesson. It is exactly after I advocate strongly in favor of #pragma once that I hit a case where it failed me.

Incredible :P